airbyte-webapp/build.gradle

```diff
@@ -106,16 +106,8 @@ task copyNginx(type: Copy) {
     into "build/docker/bin/nginx"
 }
 
-task stylelint(type: NpmTask) {
```

Comment:
:information_source: Stylelint now runs as part of the Vite (via the checker plugin). So it doesn't need a separate Gradle task anymore.